volunteer.markdown

```diff
@@ -17,7 +17,7 @@ Join the conversation on [Slack](http://slack.hotosm.org/)! A large number of vo
 
 As a mapping volunteer, you can get involved with projects around the world, and contribute to humanitarian mapping through OpenStreetMap. Even if you are a new mapper, anyone can use a computer to create basemaps by drawing over buildings and roads, or use a phone to swipe and tap population hotspots in the MapSwipe app. If you have never mapped before and would like to help, visit the Tasking Manager's [Learn page](https://tasks.hotosm.org/learn) to get started mapping in less than 30 minutes.
 
-<a href="https://tasks.hotosm.org/learn" class="btn btn-primary btn-lg btn-chevron">Learn how to map</a>
+<a href="https://tasks.hotosm.org/learn" class="btn btn-hover btn-primary btn-lg btn-chevron">Learn how to map</a>
```

Comment:
If we change this with :- <a href="https://tasks.hotosm.org/learn" class="btn btn-hover btn-primary btn-lg btn-chevron"><color : white>Learn how to map</color></a>